### PR TITLE
#35795 Pager - element update.

### PR DESCRIPTION
**Pull request by kruchy8**
PR: https://github.com/SnowdogApps/magento2-alpaca-components/issues/90
Redmine: https://dev.snowdog.pro/issues/35795

### CODE REVIEW

File: components/02-elements/pager/_pager.scss [hunk 1]
```diff
@@ -1,46 +1,61 @@
...
-$pager__item-size               : 48px !default;
-$pager__item-color--current     : $white !default;
-$pager__item-background--current: $color-primary !default;
-$pager__link-color              : $gray !default;
-$pager__link-color-hover        : $color-primary !default;
-$pager__icon-width              : 12px !default;
-$pager__icon-height             : 12px !default;
+$pager__list-padding                        : $spacer !default;
+$pager__item-margin                         : 0 $spacer;
+$pager__item--current-after-height          : $spacer;
```
Comment (kjugi) [inline components/02-elements/pager/_pager.scss]: shouldn't be `$pager__item-height-after--current` ?

File: components/02-elements/pager/pager.hbs [hunk 2]
```diff
@@ -1,14 +1,12 @@
...
-            <li class="pager__item pager__item--prev">
-                <a href="#" class="pager__link">
-                    <svg class="pager__icon">
-                        <use xlink:href="{{ svg 'arrow-left' }}"></use>
-                    </svg>
-                </a>
-            </li>
-        {{/if}}
+        <li class="pager__item">
+            <a href="#" class="pager__link pager__link--prev" {{#unless arrowLeftActive}}disabled{{/unless}}>
```
Comment (kjugi) [inline components/02-elements/pager/pager.hbs]: formatting

File: components/02-elements/pager/pager.hbs [hunk 3]
```diff
@@ -17,21 +15,19 @@
...
-            <li class="pager__item pager__item--next">
-                <a href="#" class="pager__link">
-                    <svg class="pager__icon">
-                        <use xlink:href="{{ svg 'arrow-right' }}"></use>
-                    </svg>
-                </a>
-            </li>
-        {{/if}}
+        <li class="pager__item">
+            <a href="#" class="pager__link pager__link--next" {{#unless arrowRightActive}}disabled{{/unless}}>
```
Comment (kjugi) [inline components/02-elements/pager/pager.hbs]: formatting too

File: components/02-elements/pager/pager.hbs [hunk 4]
```diff
@@ -17,21 +15,19 @@
...
+                <svg class="pager__icon">
```
Comment (kjugi) [inline components/02-elements/pager/pager.hbs]: do we have any aria-labels here?

File: components/02-elements/pager/pager.hbs [hunk 5]
```diff
@@ -1,14 +1,12 @@
...
-                <a href="#" class="pager__link">
-                    <svg class="pager__icon">
-                        <use xlink:href="{{ svg 'arrow-left' }}"></use>
-                    </svg>
-                </a>
-            </li>
-        {{/if}}
+        <li class="pager__item">
+            <a href="#" class="pager__link pager__link--prev" {{#unless arrowLeftActive}}disabled{{/unless}}>
+                <svg class="pager__icon">
```
Comment (kjugi) [inline components/02-elements/pager/pager.hbs]: what about aria-labels?

File: components/02-elements/pager/_pager.scss (hunk 1 above)
Comment (kruchy8) [reply components/02-elements/pager/_pager.scss]: I think that after should be first so I've changed it to : `$pager__item-after-height--current` what do you think ?

File: components/02-elements/pager/pager.hbs (hunk 2 above)
Comment (kruchy8) [reply components/02-elements/pager/pager.hbs]: Done.

File: components/02-elements/pager/pager.hbs (hunk 3 above)
Comment (kruchy8) [reply components/02-elements/pager/pager.hbs]: Done.

File: components/02-elements/pager/pager.hbs (hunk 4 above)
Comment (kruchy8) [reply components/02-elements/pager/pager.hbs]: Included, thanks

File: components/02-elements/pager/pager.hbs (hunk 5 above)
Comment (kruchy8) [reply components/02-elements/pager/pager.hbs]: Done.

File: components/02-elements/pager/_pager.scss (hunk 1 above)
Comment (kjugi) [reply components/02-elements/pager/_pager.scss]: actually, i'm not sure about our examples of variables 😁 If you think it's ok, keep your variable

File: components/02-elements/pager/_pager.scss [hunk 6]
```diff
@@ -1,46 +1,61 @@
...
-            color: $pager__item-color--current;
-            background-color: $pager__item-background--current;
+            position: relative;
+            &:after {
+                content: '';
+                position: absolute;
+                bottom: 14px;
+                left: 0;
+                width: 100%;
+                height: $pager__item-after-height--current ;
```
Comment (kjugi) [inline components/02-elements/pager/_pager.scss]: one space too much

File: components/02-elements/pager/_pager.scss [hunk 7]
```diff
@@ -1,46 +1,61 @@
...
+                background-color: $pager__item-after-background-color--current;
+                z-index: -1;
+                transition: $pager__item-after-transition--current ;
```
Comment (kjugi) [inline components/02-elements/pager/_pager.scss]: and here too

File: components/02-elements/pager/_pager.scss (hunk 6 above)
Comment (kruchy8) [reply components/02-elements/pager/_pager.scss]: Done

File: components/02-elements/pager/_pager.scss (hunk 7 above)
Comment (kruchy8) [reply components/02-elements/pager/_pager.scss]: Done

**Pull request merged**